common/packages/tinyscheme/scheme.c

```diff
@@ -1175,6 +1175,7 @@ static char *store_string(scheme * sc, int len_str, const char *str,
     }
     if (str != 0) {
 	snprintf(q, len_str + 1, "%s", str);
+        q[len_str] = 0;
```

Comment:
Should we add 
/*
    The original source code has been modified to be used with Private Data Objects (PDOs).
*/
at the beginning of the file?